### PR TITLE
BibAuthorID: guard unsuccesful recids

### PR DESCRIPTION
**Pull request by MSusik**
Signed-off-by: Mateusz Susik mateusz.susik@cern.ch

### CODE REVIEW

**Comment by kaplun** [comment]
What this code is supposed to do?

**Comment by MSusik** [comment]
Check error 670 on sentry. There is a variable `unsuccessful_recids`, which is a list. The error is caused by the fact that we try to access invalid key in the dictionary. I hope that author's intention was to set this variable as empty in such case.

**Comment by kaplun** [comment]
Thanks, Can you add this explanation in the commit log?

File: modules/bibauthorid/lib/bibauthorid_webinterface.py [hunk 1]
```diff
@@ -3453,7 +3453,10 @@ def commit(self, req, form):
         # Syncing is done here. Entries that have been handled are removed from
         # unsuccessful_tickets so that they do not reappear in the next reload.
         if pinfo['autoclaim']['res']:
-            unsuccessful_recids = pinfo['autoclaim']['res']['unsuccessful_recids']
+            if 'unsuccessful_recids' in pinfo['autoclaim']['res']:
+                unsuccessful_recids = pinfo['autoclaim']['res']['unsuccessful_recids']
```
Comment (kaplun) [inline modules/bibauthorid/lib/bibauthorid_webinterface.py:3457]: Actually you can write:

``` python
unsuccessful_recids = pinfo['autoclaim']['res'].get('unsuccessful_recids', [])
```

**Pull request merged**